playground/src/specs/circular.ts

```diff
@@ -0,0 +1,141 @@
+export const circular = `
```

Comment:
Where is this used?

Reply:
It's used in playground. If you develop you can choose schema and it's only used to check that circular (very complex) schemas are also supported.